build.sbt

```diff
@@ -21,16 +20,18 @@ resolvers ++= Seq(
 
 libraryDependencies ++= Seq(
   "org.webjars" % "swagger-ui" % "4.1.3",
-  "org.broadinstitute.dsde" %%  "rawls-model" % rawlsModelV
```

Comment:
I got rid of the rawls-model junk too. Much cleaner to just use the existing workbench-libs implementations for this.